Services/TermsOfService/classes/class.ilTermsOfServiceAcceptanceStartUpStep.php

```diff
@@ -51,7 +51,7 @@ public function shouldInterceptRequest() : bool
 
             /** @var ilTermsOfServiceHelper $tosService */
             $tosService = $this->dic['tos.service'];
-            if ($tosService->hasToResignAcceptance($this->dic->user(), $this->dic->logger()->tos())) {
+            if ($tosService->hasToResignAcceptance($this->dic->user(), $this->dic->logger()->tos())) {  // PHP8-Review: Method 'tos' not found in ILIAS\DI\LoggingServices
```

Comment:
This is a `False Positive`. There is a magic `__call` method in the logger component which returns a logger based on the component, here `tos`.